src/Form/EventListener/AnyNoneFilterListener.php

```diff
@@ -61,19 +61,21 @@ public function __construct(ColumnsExtractor $columnsExtractor)
     }
 
     /**
-     * Adds data for AnyNoneFilter to the form (events, ...).
+     * Adds data for AnyNoneFilter to the form (events, fields, ...).
      *
      * @param \CubeTools\CubeCommonBundle\Form\EventListener\FormBuilderInterface $builder
      */
     public function addToBuilder(FormBuilderInterface $builder)
     {
-        $builder->addEventListener(FormEvents::PRE_SET_DATA, array($this, 'addAnyNoneColumns'));
+        $this->addAnyNoneFields($builder);
```

Comment:
pro doing here instead of an event:
*
contra:
*

Reply:
exactly, by using PRE_SET_DATA we are not dependent from place of execution of addToBuilder method

Reply:
fixed